module/input/Camera/src/Camera.cpp

```diff
@@ -357,7 +413,7 @@ namespace input {
     }
 
     void Camera::control_lost(ArvGvDevice*) {
-        NUClear::log<NUClear::WARN>("Control of a camera has been lost ");
+        NUClear::log<NUClear::FATAL>("Control of a camera has been lost ");
```

Comment:
Should still add in the context pointer

Reply:
Added, and added the name to the message